tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -82,6 +85,31 @@ Verify NFS Volume Already Created
     Should Be Equal As Integers  ${rc}  1
     Should Contain  ${output}  Error response from daemon: A volume named ${containerVolName} already exists. Choose a different volume name.
 
+Reboot VM and Verify Basic VCH Info
+    Log To Console  Rebooting VCH\n - %{VCH-NAME}
+    Reboot VM  %{VCH-NAME}
+
+    Log To Console  Getting VCH IP ...
+    ${new_vch_ip}=  Get VM IP  %{VCH-NAME}
+    Log To Console  New VCH IP is ${new_vch_ip}
+    Replace String  %{VCH-PARAMS}  %{VCH-IP}  ${new_vch_ip}
```

Comment:
@mhagen-vmware I followed an example to implement this.  I'll find that other instance and fix it as well.

- [ ] Use `Replace String` as intended.
- [ ] Fix other instance of `Replace String`